benchmarks/bytes_and_flops/main.cpp

```diff
@@ -105,17 +119,26 @@ int main(int argc, char* argv[]) {
     return 0;
   }
 
+  if ((B != 2) && (B != 10)) {
+    printf("B must be one of 2,10\n");
+    return 0;
```

Comment:
I love that we return success error code.  (not asking you to change I see it is consistent with what was there before)